### PR TITLE
Report words used across files

### PR DESCRIPTION
**Pull request by cston**
Command-line tool to report a sorted list of words used across files in a source directory.
If the word appears once, the file and location are included; if the word appears multiple times all within one file, the file is included.

Used as a simple mechanism for catching typos.

### CODE REVIEW

**Comment by cston** [comment]
@dotnet/roslyn-infrastructure please review.

File: Roslyn.sln [hunk 1]
```diff
@@ -359,6 +359,8 @@ Project("{778DAE3C-4631-46EA-AA77-85C1314464D9}") = "BasicCodeStyleTests", "src\
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RoslynPublish", "src\Tools\RoslynPublish\RoslynPublish.csproj", "{2D36C343-BB6A-4CB5-902B-E2145ACCB58F}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Words", "src\Tools\Source\Words\Words.csproj", "{D3B5A8BE-2249-45E6-A7F9-70CBBDE6A405}"
```
Comment (jasonmalinowski) [inline Roslyn.sln]: Should this be in roslyn-tools?

**Comment by cston** [comment]
@jasonmalinowski, I wasn't aware of the roslyn-tools repo but I agree it makes more sense there. I'll create a PR in that repo.

**Comment by cston** [comment]
Created https://github.com/dotnet/roslyn-tools/pull/59.

**Pull request closed without merging**